app/actors/curation_concerns/actors/generic_work_actor.rb

```diff
@@ -1,6 +1,6 @@
 # Generated via
 #  `rails generate curation_concerns:work GenericWork`
-module CurationConcerns
+module CurationConcerns::Actors
```

Comment:
Any reason we disabled `Style/ClassAndModuleChildren` instead of addressing the issue?